tools/shell/embedded_shell.cpp

```diff
@@ -198,11 +207,20 @@ void EmbeddedShell::run() {
         } else if (lineStr.rfind(shellCommand.SHOW_REL) == 0) {
             printRelSchema(lineStr.substr(shellCommand.SHOW_REL.length()));
         } else if (!lineStr.empty()) {
-            auto queryResult = conn->query(lineStr);
-            if (queryResult->isSuccess()) {
-                printExecutionResult(*queryResult);
-            } else {
-                printf("Error: %s\n", queryResult->getErrorMessage().c_str());
+            stringstream ss(lineStr);
+            string thisQuery;
```

Comment:
I think you can move these two vars out of the while loop, so we don't need to create them over and over again in this while loop. Also rename thisQuery -> query